packages/subgraph/src/mappingHelpers.ts

```diff
@@ -246,6 +282,7 @@ export function getOrInitStream(event: FlowUpdated): Stream {
         stream.streamedUntilUpdatedAt = BigInt.fromI32(0);
         stream.updatedAtTimestamp = currentTimestamp;
         stream.updatedAtBlockNumber = event.block.number;
+        stream.userData = Bytes.empty();
```

Comment:
Hmm, why not use the `userData` from the event?

Reply:
true